src/main/java/core/basesyntax/Machines/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.Machines;
+
+public class Truck extends Machine{
+  @Override
+  public void doWork() {
+    System.out.println("Truck is shipping cargo.");
+  }
+  @Override
+  public void stopWork() {
+    System.out.println("Truck has finished it's work");
```

Comment:
The message in `System.out.println` should be more informative and indicate the type of Machine. For example: `Truck stopped its work.` Also, avoid using contractions like `it's` in formal code comments or messages; use `it has` instead.